src/RSLoad/PowerBILoad.loadtest

```diff
@@ -0,0 +1,1662 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LoadTest Name="PowerBILoad" Description="" Owner="" storage="e:\git\public\reporting-services-loadtest\src\rsload\powerbiload.loadtest" Priority="2147483647" Enabled="true" CssProjectStructure="" CssIteration="" DeploymentItemsEditable="" WorkItemIds="" TraceLevel="None" CurrentRunConfig="RS Portal_Classic load " Id="fa3a9933-2dad-48f3-8a74-534d75aa1311" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
+  <Scenarios>
+    <Scenario Name="PowerBI_Reports" DelayBetweenIterations="0" PercentNewUsers="0" IPSwitching="false" TestMixType="PercentageOfTestsStarted" ApplyDistributionToPacingDelay="true" MaxTestIterations="0" DisableDuringWarmup="false" DelayStartTime="0" AllowedAgents="">
+      <ThinkProfile Value="0.2" Pattern="NormalDistribution" />
+      <LoadProfile Pattern="Constant" InitialUsers="60" />
+      <TestMix>
+        <TestProfile Name="UsePowerBIReportsEmbeddedReuseModel" Path="bin\debug\rsload.dll" Id="78953d39-c69d-0c9e-2601-1fef9ea8926a" Percentage="25" Type="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestElement, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.ObjectModel, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+        <TestProfile Name="UsePowerBIReportsEmbeddedStreamNewModel" Path="bin\debug\rsload.dll" Id="5d79ed0b-4372-43e2-bfc2-30387c73a685" Percentage="25" Type="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestElement, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.ObjectModel, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
We probably want to reduce the weight of "UsePowerBIReportsEmbeddedStreamNewModel" to 5.